### PR TITLE
kdump-ignition test added to verify kdump works via ignition config

### PR DESCRIPTION
**Pull request by aaradhak**

### CODE REVIEW

**Comment by aaradhak** [comment]
Hitting this test failure on running the kola-igntion test

```
[coreos-assembler]$ cosa kola run -p qemu --qemu-image /srv/fedora-coreos-config/fedora-coreos-35.20211203.2.1-qemu.x86_64.qcow2 -E /srv/fedora-coreos-config/ ext.fedora-coreos-config.kdump.kdump-ignition
kola --output-dir tmp/kola run -p qemu --qemu-image /srv/fedora-coreos-config/fedora-coreos-35.20211203.2.1-qemu.x86_64.qcow2 -E /srv/fedora-coreos-config/ ext.fedora-coreos-config.kdump.kdump-ignition
=== RUN   ext.fedora-coreos-config.kdump.kdump-ignition
systemctl status kola-runext.service:
× kola-runext.service
     Loaded: loaded (/etc/systemd/system/kola-runext.service; static)
     Active: failed (Result: exit-code) since Wed 2022-01-26 14:20:23 UTC; 82ms ago
    Process: 2053 ExecStart=/usr/local/bin/kola-runext-test.sh (code=exited, status=1/FAILURE)
   Main PID: 2053 (code=exited, status=1/FAILURE)
        CPU: 3ms

Jan 26 14:20:23 qemu0 systemd[1]: Started kola-runext.service.
Jan 26 14:20:23 qemu0 kola-runext-test.sh[2053]: + case "${AUTOPKGTEST_REBOOT_MARK:-}" in
Jan 26 14:20:23 qemu0 kola-runext-test.sh[2054]: ++ find /var/crash -type f -name vmcore
Jan 26 14:20:23 qemu0 kola-runext-test.sh[2053]: + kcore=
Jan 26 14:20:23 qemu0 kola-runext-test.sh[2053]: + test -z ''
Jan 26 14:20:23 qemu0 kola-runext-test.sh[2053]: + fatal 'No kcore found in /var/crash'
Jan 26 14:20:23 qemu0 kola-runext-test.sh[2053]: + echo 'No kcore found in /var/crash'
Jan 26 14:20:23 qemu0 kola-runext-test.sh[2053]: No kcore found in /var/crash
Jan 26 14:20:23 qemu0 kola-runext-test.sh[2053]: + exit 1
Jan 26 14:20:23 qemu0 systemd[1]: kola-runext.service: Main process exited, code=exited, status=1/FAILURE
Jan 26 14:20:23 qemu0 systemd[1]: kola-runext.service: Failed with result 'exit-code'.
--- FAIL: ext.fedora-coreos-config.kdump.kdump-ignition (56.08s)
        harness.go:901: kolet failed: : kolet run-test-unit failed: Error: Unit kola-runext.service exited with code 1
2022-01-26T14:20:23Z cli: Unit kola-runext.service exited with code 1: Process exited with status 1
) on machine ba362888-54d0-4ca5-9b4b-b09bc6d36c0c consolered crash
FAIL, output in tmp/kola
Error: harness: test suite failed
2022-01-26T14:20:26Z cli: harness: test suite failed
```

**Comment by travier** [comment]
See https://github.com/coreos/fedora-coreos-tracker/issues/1079#issuecomment-1033577657

File: tests/kola/kdump/kdump-ignition/config.bu [hunk 1]
```diff
@@ -0,0 +1,10 @@
+variant: fcos
+version: 1.4.0
+kernel_arguments:
+  should_exist:
+  - 'crashkernel=256M'
+systemd:
+  units:
+  - enabled: true
+    name: kdump.service
+
```
Comment (travier) [inline tests/kola/kdump/kdump-ignition/config.bu:10]: Missing new lines at EOF (both files). You should probably configure your editor to add them automatically.

**Comment by dustymabe** [comment]
As of https://github.com/coreos/fedora-coreos-config/commit/9102625bcc880458f91db7d0b4dbc232cd6cd6d1 the `ext.config.kdump.crash` test now sets up kdump via Ignition and tests that it works.

**Pull request closed without merging**